eng/config/PublishData.json

```diff
@@ -130,7 +130,7 @@
         "Shipping",
         "NonShipping"
       ],
-      "vsBranch": "main",
```

Comment:
probably also change main-vs-deps to create non-draft PRs (can't comment on that line)

Reply:
Sure, what about 17.5p1?

Reply:
i think 17.5p1 should still be draft (since its qb)